src/components/MainTitle.vue

```diff
@@ -16,7 +16,18 @@ export default {
       type: String,
       required: false,
     },
+    positionType: {
+      type: String,
```

Comment:
Setting either `default` or `required` keys on props object is usually a good idea. It will describe the prop more accurately. In this case, you can go with the default value, so you can omit to pass the prop entirely for some most common button types (e.g. primary, left, etc.).